tarql/places/place-types-10-21-24.csv

```diff
@@ -0,0 +1,18 @@
+TOL ID,BDRC ID,Wikidata ID,Site (Place) Type,Wylie,Wylie Variant,Tibetan,Tibetan Variant,Chinese,Class,Definition,Notes
```

Comment:
I'll delete this file. It's used to build a taxonomy and thus goes in the tol-ontology repo.